topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md

```diff
@@ -0,0 +1,449 @@
+
+The knee and inflection points on the curve mark the transition between two components of the total count distribution. This is assumed to represent the difference between empty droplets with little RNA and cell-containing droplets with much more RNA, and gives us a rough idea of how many cells to expect in our sample.
+
+> ### Question {% icon question %} Questions
+>
+> 1. How many cells do we expect to see in our sample based on the above plot?
+> 1. What is the threshold of total UMIs to which a cell can be considered to be empty?>
+>
+> > ### {% icon solution %} Solution
+> > 1. We see the blue knee line cross the threshold of barcodes at just greater than the 100 Rank on the horizontal log scale, so we can assume that our assessment that there should be around 300 cells is correct.
+> > 1. The threshold is given by the inflection line which tells us that any cell with a total UMI count less than 3 should be considered empty.
+> >   **However**, this dataset has been greatly sub-sampled for training purposes (~1000 down to ~300 cells) so the sensitivity of this value should be taken into account. Due to the already low number of cells, we should realistically set this to 1.
+> {: .solution}
+{: .question}
+
+On large 10x datasets we can use these thresholds as metrics to utilise in our own custom filtering, which is once again provided by the **DropletUtils** tool.
+
+> ### {% icon hands_on %} Hands-on: Custom Filtering
+>
+> **DropletUtils** {% icon tool %} with the following parameters:
+>    - *"Format for the input matrix"*: `Bundled (barcodes.tsv, genes.tsv, matrix.mtx)`
+>        - {% icon param-file %} *"Count Data"*: `output_matrix` (output of **RNA STARsolo** {% icon tool %})
+>        - {% icon param-file %} *"Genes List"*: `output_genes` (output of **RNA STARsolo** {% icon tool %})
+>        - {% icon param-file %} *"Barcodes List"*: `output_barcodes` (output of **RNA STARsolo** {% icon tool %})
+>    - *"Operation"*: `Filter for Barcodes`
+>        - *"Method"*: `EmptyDrops`
+>            - *"Lower-bound Threshold"*: `1`
+>            - *"FDR Threshold"*: `0`
+>        - *"Format for output matrices"*: `Tabular`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > We set the lower-bound to 1 instead of 3 due to the sparsity of the dataset. On a full dataset, you would insert the value you discovered using the Rank method.
+>    >
+>    > The False Discovery Rate (FDR) is set to zero because the ambient profile is too weak to measure against on this dataset.
+>    >
+>    {: .comment}
+>
+{: .hands_on}
+
+![cells]({{ site.baseurl }}{% link topics/transcriptomics/images/tenx_cells.png %} "Detected Cells (red)")
+
+
+Here we recover 137 high quality cells instead of the detected 272 given previously. On large datasets, this difference can help clean downstream clustering which may appear softer and less well-defined on a more inclusive dataset than that which has undergone tighter quality controls.
```

Comment:
Can we split the second sentence into two and make the sentence easier to understand and therefore to translate?